test/cqlpy/suite.yaml

```diff
@@ -5,4 +5,5 @@ dirties_cluster:
 extra_scylla_cmdline_options:
   - '--experimental-features=udf'
   - '--experimental-features=keyspace-storage-options'
+  - '--experimental-features=views-with-tablets'
```

Comment:
I see you also added this option to test/pylib/scylla_cluster.py - so I don't think it's needed again in test/cqlpy/suite.yaml.

Reply:
I tried running tests and seems like the cqlpy tests still need it, at least when using ./test.py in release mode (interestingly, they also fail when the feature is missing in `test/pylib/scylla_cluster.py` instead)